docs/frameworks/laravel.md

```diff
@@ -111,6 +111,42 @@ If your application creates links and redirections to HTTP URLs (which are inval
 +    protected $proxies = '*';
 ```
 
+### Forwarding correct Host header
+
+When not enabling host forwarding, Laravel will use the hostname/domain from Cloudfront while building internal links (routes).
+To fix this, you can use one of the following methods. Please also have a look at the HTTPS section above for allowing the AWS proxies.
+
+#### Method 1: Extending serverless.yml
+
+You can add the following `OriginCustomHeaders` part to your `Origins` setting in your `serverless.yml` file:
+
+```diff
+Origins:
+  # The website (AWS Lambda)
+  - Id: Website
+    DomainName: '#{ApiGatewayRestApi}.execute-api.#{AWS::Region}.amazonaws.com'
+    OriginPath: '/dev'
+    CustomOriginConfig:
+      OriginProtocolPolicy: 'https-only' # API Gateway only supports HTTPS
++    OriginCustomHeaders:
++      - HeaderName: 'X-Forwarded-Host'
++        HeaderValue: 'your-domain-here.com'
+```
+
+#### Method 2: Customizing the Cloudfront Distribution (whitelisting `Host` header forwarding)
+
+You can whitelist the `Host` header in your Cloudfront Distribution like this (to forward the `Host` header from the request):
+
+- Open the Amazon CloudFront console, and then choose your distribution.
+- Choose the Behaviors view, and then choose the path you are using.
+- Choose Edit.
+- For Cache Based on Selected Request Headers, choose Whitelist.
+- Under Whitelist Headers, choose Host from the column on the left, and then choose Add.
+- Choose Yes, Edit.
+
+*Origin: https://aws.amazon.com/de/premiumsupport/knowledge-center/configure-cloudfront-to-forward-headers/*
```

Comment:
Unfortunately this breaks API Gateway (because it doesn't know which API Gateway to call, see https://stackoverflow.com/a/42259852/245552). So this is not an option we can use 😞

Reply:
That's odd, I've got this working with API Gateway v2 and Host header forwarding for my Laravel app.

Reply:
That is very weird indeed, can you double check? I tried it but it doesn't work for me. Are you simply whitelisting the "Host" header (so that it's forwarded by CloudFront to API Gateway)?